bin/qless-js-worker.js

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env node
+
+const assert = require('assert');
+const qless = require('../index.js');
+
+const commander = require('commander');
+
+function collect(name, queues) {
+  queues.push(name);
+  return queues;
+}
+
+function increaseVerbosity(v, total) {
+  return total + 1;
+}
+
+commander
+  .option('-h, --host <url>', 'The redis:// url to connect to')
```

Comment:
I wonder if this would be clearer if it were specified as `--redis` instead of `--host`. I know that probably follows the pattern of other qless workers, but I wonder if this was an original oversight that should be remedied.